nmostesting/mocks/Registry.py

```diff
@@ -177,6 +282,31 @@ def check_authorized(self, headers, path, write=False):
 REGISTRIES = [Registry(REGISTRY_COMMON, i + 1) for i in range(NUM_REGISTRIES)]
 REGISTRY_API = Blueprint('registry_api', __name__)
 
+@REGISTRY_API.route('/x-nmos', methods=["GET"], strict_slashes=False)
+def x_nmos():
+    registry = REGISTRIES[flask.current_app.config["REGISTRY_INSTANCE"]]
+    if not registry.enabled:
+        abort(503)
+    authorized = registry.check_authorized(request.headers, request.path)
```

Comment:
I don't believe this path requires an authorization check (https://specs.amwa.tv/is-10/releases/v1.0.0/docs/4.5._Behaviour_-_Resource_Servers.html#path-validation).

Reply:
Agreed